[2022-04-07 19:25:51,436] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2018-01-01T00:00:00+00:00 [queued]>
[2022-04-07 19:25:51,592] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2018-01-01T00:00:00+00:00 [queued]>
[2022-04-07 19:25:51,592] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:25:51,592] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 19:25:51,593] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 19:25:52,760] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task> on 2018-01-01 00:00:00+00:00
[2022-04-07 19:25:52,828] {standard_task_runner.py:52} INFO - Started process 4372 to run task
[2022-04-07 19:25:52,882] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task', 'scheduled__2018-01-01T00:00:00+00:00', '--job-id', '12965', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmp75goyjr2', '--error-file', '/tmp/tmp3wn_p49_']
[2022-04-07 19:25:52,920] {standard_task_runner.py:80} INFO - Job 12965: Subtask create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task
[2022-04-07 19:25:53,439] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2018-01-01T00:00:00+00:00 [running]> on host 2c54f2187010
[2022-04-07 19:25:53,908] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2018-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2018-01-01T00:00:00+00:00
[2022-04-07 19:25:53,910] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 19:25:53,966] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 19:25:54,043] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_badges_bq_create_badges_stack_overflow_data_partitioned_table_task_2018_01_01T00_00_00_00_00_5e02ea611d6977cb6722ffda86bd6d05
[2022-04-07 19:25:54,657] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2153, in execute
    job = self._submit_job(hook, job_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2119, in _submit_job
    retry=self.result_retry,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1560, in insert_job
    job.result(timeout=timeout, retry=retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1498, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/query.py", line 1488, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 135, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Could not cast literal "date" to type TIMESTAMP at [1:102]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_badges_bq_create_badges_stack_overflow_data_partitioned_table_task_2018_01_01T00_00_00_00_00_5e02ea611d6977cb6722ffda86bd6d05

[2022-04-07 19:25:54,769] {taskinstance.py:1288} INFO - Marking task as FAILED. dag_id=load_to_BigQuery, task_id=create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task, execution_date=20180101T000000, start_date=20220407T192551, end_date=20220407T192554
[2022-04-07 19:25:54,865] {standard_task_runner.py:98} ERROR - Failed to execute job 12965 for task create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task (400 Could not cast literal "date" to type TIMESTAMP at [1:102]

Location: us-central1
Job ID: airflow_load_to_BigQuery_create_external_badges_bq_create_badges_stack_overflow_data_partitioned_table_task_2018_01_01T00_00_00_00_00_5e02ea611d6977cb6722ffda86bd6d05
; 4372)
[2022-04-07 19:25:54,957] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-04-07 19:25:56,420] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-07 22:18:52,566] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2018-01-01T00:00:00+00:00 [queued]>
[2022-04-07 22:18:52,644] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2018-01-01T00:00:00+00:00 [queued]>
[2022-04-07 22:18:52,644] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:18:52,644] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 22:18:52,644] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:18:53,187] {taskinstance.py:1270} INFO - Executing <Task(BigQueryInsertJobOperator): create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task> on 2018-01-01 00:00:00+00:00
[2022-04-07 22:18:53,225] {standard_task_runner.py:52} INFO - Started process 6672 to run task
[2022-04-07 22:18:53,255] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task', 'scheduled__2018-01-01T00:00:00+00:00', '--job-id', '14225', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmpqmpa9k2p', '--error-file', '/tmp/tmpj3ychvpc']
[2022-04-07 22:18:53,277] {standard_task_runner.py:80} INFO - Job 14225: Subtask create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task
[2022-04-07 22:18:53,587] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task scheduled__2018-01-01T00:00:00+00:00 [running]> on host 36107d30f64e
[2022-04-07 22:18:53,857] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task
AIRFLOW_CTX_EXECUTION_DATE=2018-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2018-01-01T00:00:00+00:00
[2022-04-07 22:18:53,859] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 22:18:53,885] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 22:18:53,935] {bigquery.py:1554} INFO - Inserting job ***_load_to_BigQuery_create_external_badges_bq_create_badges_stack_overflow_data_partitioned_table_task_2018_01_01T00_00_00_00_00_930d3aa0ffccdfd5cdcd0f2fb8b6d659
[2022-04-07 22:18:56,682] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=load_to_BigQuery, task_id=create_external_badges.bq_create_badges_stack_overflow_data_partitioned_table_task, execution_date=20180101T000000, start_date=20220407T221852, end_date=20220407T221856
[2022-04-07 22:18:56,799] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-07 22:18:56,868] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
